config/production.json

```diff
@@ -9,6 +9,9 @@
 			"applicationId": "4486817b-44b8-4e6c-b6a4-cef83e48cf46",
 			"clientToken": "pub750ceadc5a9c79a6d3da59b534eb5108",
 			"service": "developer.hashicorp.com"
+		},
+		"auth": {
+			"idp_url": "https://auth.idp.hashicorp.com"
```

Comment:
Question 🙋🏻 Do we need a `session_max_age` here like is being set in `config/base.json`? Or is there a default value being used that we're happy with in prod?

Reply:
For now we'll have this inherit from `base.json`, but I expect we'll adjust this for prod at some point!

Reply:
Ah, I see! I didn't realize we were merging nested properties of these config files already. 😊